core-bundle/src/Cron/LegacyCron.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Cron;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\CronJob;
+use Contao\System;
+use Terminal42\ServiceAnnotationBundle\ServiceAnnotationInterface;
+
+class LegacyCron implements ServiceAnnotationInterface
+{
+    private const ALLOWED_CRONS = [
```

Comment:
As discussed, we do not need exceptions for the core cron jobs.

Reply:
Have we decided this yet? 😁 

If we only implement the core cronjobs as services in Contao `4.10`, then the LTS version will always trigger deprecation warnings, since the core still uses `TL_CRON` there.

Reply:
Yes, this will be the case. Just like it will be the case for every third-party extension that defines cron jobs. 🤷‍♂